dev-java/java-service-wrapper/java-service-wrapper-3.5.50.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source test"
+inherit java-pkg-2 java-ant-2 toolchain-funcs
+
+MY_PN="wrapper"
+MY_P="${MY_PN}_${PV}_src"
+DESCRIPTION="A wrapper that makes it possible to install a Java Application as daemon"
+HOMEPAGE="https://wrapper.tanukisoftware.com/"
+SRC_URI="https://${MY_PN}.tanukisoftware.com/download/${PV}/${MY_P}.tar.gz"
+
+LICENSE="tanuki-community"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+RDEPEND="
+	virtual/jre:1.8"
```

Comment:
>=virtual/jre-1.8:*"

Reply:
These don't seem to work:
```
dev-java/java-service-wrapper
  NonsolvableDepsInDev: version 3.5.50: nonsolvable depset(depend) keyword(~amd64) dev profile (default/linux/amd64/17.0/x32) (32 total): solutions: [ =virtual/jdk-1.8:* ]
  NonsolvableDepsInDev: version 3.5.50: nonsolvable depset(rdepend) keyword(~amd64) dev profile (default/linux/amd64/17.0/x32) (32 total): solutions: [ =virtual/jre-1.8:* ]
  NonsolvableDepsInStable: version 3.5.50: nonsolvable depset(depend) keyword(~amd64) stable profile (default/linux/amd64/17.1) (27 total): solutions: [ =virtual/jdk-1.8:* ]
  NonsolvableDepsInStable: version 3.5.50: nonsolvable depset(rdepend) keyword(~amd64) stable profile (default/linux/amd64/17.1) (27 total): solutions: [ =virtual/jre-1.8:* ]
```

Reply:
> These don't seem to work:
> 
> ```
> dev-java/java-service-wrapper
>   NonsolvableDepsInDev: version 3.5.50: nonsolvable depset(depend) keyword(~amd64) dev profile (default/linux/amd64/17.0/x32) (32 total): solutions: [ =virtual/jdk-1.8:* ]
>   NonsolvableDepsInDev: version 3.5.50: nonsolvable depset(rdepend) keyword(~amd64) dev profile (default/linux/amd64/17.0/x32) (32 total): solutions: [ =virtual/jre-1.8:* ]
>   NonsolvableDepsInStable: version 3.5.50: nonsolvable depset(depend) keyword(~amd64) stable profile (default/linux/amd64/17.1) (27 total): solutions: [ =virtual/jdk-1.8:* ]
>   NonsolvableDepsInStable: version 3.5.50: nonsolvable depset(rdepend) keyword(~amd64) stable profile (default/linux/amd64/17.1) (27 total): solutions: [ =virtual/jre-1.8:* ]
> ```

Sorry, it should read `>=virtual/jdk-1.8:*` and `>=virtual/jre-1.8:*`
as described in https://wiki.gentoo.org/wiki/Java_Developer_Guide#Dependencies